rmf_site_format/src/anchor.rs

```diff
@@ -41,16 +42,16 @@ impl From<[f32; 2]> for Anchor {
     }
 }
 
-fn to_slice(p: &[f32]) -> &[f32; 2] {
-    p.try_into().expect("Wrong array size")
+fn to_slice(p: &[f32; 3]) -> [f32; 2] {
```

Comment:
It would be inaccurate to keep calling this `to_slice` since the term "slice" refers to something of the form `&[T]`, and we no longer have the `&`. A better function name might be `to_array2` or `truncate`.